Makefile

```diff
@@ -75,6 +75,15 @@ endif
 export GOOS ?= $(GOOS_LOCAL)
 
 export ENABLE_COREDUMP ?= false
+
+# Enable Istio CNI in helm template commands
+export ENABLE_ISTIO_CNI ?= false
+
+# NOTE: env var EXTRA_HELM_SETTINGS can contain helm chart override settings, example:
+# EXTRA_HELM_SETTINGS="--set istio-cni.excludeNamespaces={} --set istio-cni.tag=v0.1-dev-foo"
+
+
+ISTIO_HELM_REPO := https://raw.githubusercontent.com/istio/istio.io/master/static/charts
```

Comment:
Now that we build these charts in daily releases (and will start building in the next monthly release). Should we be pointing to those instead?

Reply:
That is probably a good idea to ensure we are testing a given CNI adapter that is compatible to an Istio build.

Reply:
@linsun - there is an issue open to track the adding of a test.  It's 9484, I don't want to directly link it here since this PR won't close or directly impact that issue.   Note separately we need to decide when and how that gets triggered.   The initial work will just enable the ability to run the test from an istio/istio workspace.

Reply:
fair point - I agree these can be tracked separately in a follow up WI.

Reply:
However I think we should probably remove these static packages built from an unknown snapshot now, given these are built by daily releases.